mtools/mlaunch/mlaunch.py

```diff
@@ -769,7 +768,7 @@ def init(self):
                 con = self.client('localhost:%i' % mongos[0])
 
                 shards_to_add = len(self.shard_connection_str)
-                nshards = con['config']['shards'].count()
+                nshards = con['config']['shards'].count_documents()
```

Comment:
I suppose this code is untested. Here and everywhere else you need to pass a filter to count_documents, ie:
```
count_documents({})
```

Reply:
Thanks!